package.json

```diff
@@ -14,9 +14,9 @@
     "coverage:report": "nyc report --check-coverage --statements 50",
     "lint": "next lint",
     "lint:fix": "next lint --fix",
-    "prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --ignore-unknown",
-    "prettier:check": "npm run prettier -l",
-    "prettier:fix": "npm run prettier --write"
+    "prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --write --ignore-unknown",
+    "prettier:check": "npx eslint-config-prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}'",
```

Comment:
As per https://github.com/prettier/eslint-config-prettier#cli-helper-tool this check for conflicts between `prettier` and `eslint` rules. The expectation is to check for formatting errors.

Reply:
Okay will change this.